libraries/AP_Scripting/lua_scripts.cpp

```diff
@@ -117,6 +117,34 @@ int lua_scripts::atpanic(lua_State *L) {
     return 0;
 }
 
+// helper for print and log of runtime stats
+void lua_scripts::update_stats(const char *name, uint32_t run_time, int total_mem, int run_mem)
+{
+    if ((_debug_options.get() & uint8_t(DebugLevel::RUNTIME_MSG)) != 0) {
```

Comment:
Ewww.

Is it time for
```suggestion
    if (debug_option_is_set(DebugLevel::RUNTIME_MSG))) {
```

... also, is this a bitmask or a "level"?!

Reply:
The problem is that `_debug_options` is a const reference to the parameter in the front end. It is used both here and in the front end. We could do a `debug_option_is_set` in the front end, but then we would have to pass a front end object in or go via the singleton.

So TLDR its more involved to add a helper than it seems. 

It was a level, we changed to a bitmask but did not rename everything.